### PR TITLE
[API] Remove API directory @open sesame 03/22 11:17

### PR DESCRIPTION
**Pull request by anyj0527**
- Remove `NNSTREAMER_CAPI_*` from jni/nnstreamer.mk
- Fix Android build guide after ML API repo
- Remove API directory

This PR finishes the migration of API related code to ML API repo (github.com/nnstreamer/api)

Migration process (Please review related PRs)
- [x] https://github.com/nnstreamer/TAOS-CI/pull/712 : remove api build in nnstreamer and new script for ml-api repo 
- [x] https://github.com/nnstreamer/nnstreamer-android-resource/pull/18 : change directory name (api > nnstreamer)
- [x] Update the CI : enable ci/android build for ml-api repo
- [x] https://github.com/nnstreamer/api/pull/18 : check the CI can check build android lib
- [ ] This PR : remove api directory

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: jni/nnstreamer.mk [hunk 1]
```diff
@@ -15,7 +15,6 @@ NNSTREAMER_VERSION_MICRO := $(word 3,$(subst ., ,${NNSTREAMER_VERSION}))
 
 NNSTREAMER_GST_HOME := $(NNSTREAMER_ROOT)/gst/nnstreamer
 NNSTREAMER_EXT_HOME := $(NNSTREAMER_ROOT)/ext/nnstreamer
-NNSTREAMER_CAPI_HOME := $(NNSTREAMER_ROOT)/api/capi
```
Comment (jaeyun-jung) [inline jni/nnstreamer.mk:18]: let's update taos-ci first.

**Comment by jaeyun-jung** [comment]
After preparing taos-ci, add another commit to remove api directory in nnstreamer repo.
This is final step for API migration :)

**Review by zhoonit — APPROVED** [review]
If android build is verified, LGTM :)

File: Documentation/getting-started-android.md [hunk 2]
```diff
@@ -235,11 +239,11 @@ You can download these files from [nnsuite testcases repository](https://github.
...
 
 To check the testcases, run the build script with an option ```--run_test=yes```.
-You can find the result in ```$NNSTREAMER_ROOT/android_lib```.
+You can find the result in ```$ML_API_ROOT/android_lib```.
 
 ```bash
-$ cd $NNSTREAMER_ROOT
-$ bash ./api/android/build-android-lib.sh --run_test=yes
+$ cd $ML_API_ROOT
+$ bash ./java/android/build-android-lib.sh --run_test=yes
```
Comment (jaeyun-jung) [inline Documentation/getting-started-android.md:246]: fix script to `build-nnstreamer-android-lib.sh`

Comment (anyj0527) [reply Documentation/getting-started-android.md]: Updated, Thanks!

**Review by jaeyun-jung — APPROVED** [review]

**Comment by jaeyun-jung** [comment]
ML API repo is prepared, now remove api directory in nnstreamer.

**Pull request merged**